rider/renovate.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended"
+    ],
+    "packageRules": [
+        {
+            "matchPackagePatterns": [
+                "org.jetbrains.kotlin.jvm",
+                "org.jetbrains.kotlin:kotlin-stdlib"
+            ],
+            "enabled": false
+        },
+        {
+            "matchPackagePatterns": [
+                "com.jetbrains.rd:rd-gen"
+            ],
+            "enabled": false,
+            "description": ["rd-gen is managed by intellij-updater."]
+        }
+    ]
+}
```

Comment:
We don't use Renovate in JetBrains-owned repositories (yet?), and my current recommendation is Dependabot. So, this config is not needed.